src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+p {
+  margin: 0;
+}
+
+.card {
+  height: 406px;
+  width: 198px;
+
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    margin: 32px auto 0;
+    height: 134px;
+    width: 160px;
+
+    background-image: url(../images/imac.jpeg);
+    background-size: 100% 100%;
+  }
+
+  &__title {
+    margin: 40px 16px 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: #060b35;
+  }
+
+  &__code {
+    margin: 4px 16px 0;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: #616070;
+  }
+
+  &__review {
+    width: 166px;
+    height: 16px;
+    margin: 16px 16px 0;
+
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    width: 166px;
+    height: auto;
+    margin: 24px 16px 0;
+
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__button {
+    width: 164px;
+    height: 38px;
+    margin: 16px;
+
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+    background-color: #00acdc;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: #fff;
+    text-transform: uppercase;
+  }
```

Comment:
The button should change its dimensions based on the padding and the text inside it. Fixed dimensions can lead to issues with responsiveness or text overflow.